composer.json

```diff
@@ -13,42 +13,42 @@
     },
     "scripts": {
         "post-install-cmd": [
-            "composer get-okta-oidc-tck",
-            "if [[ `echo npm -v` ]]; then npm install; fi",
```

Comment:
Removed this check for npm installation as it was not working on windows.
If the user doesn't have npm installed, the script will anyway fail.